ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml

```diff
@@ -59,6 +65,32 @@ images:
       - destination_dir: .
         source_path: /go/src/github.com/stackrox/stackrox/docs/.
   to: docs
+- build_args:
+  - name: GO_VERSION
+    value: 1.17.2
+  - name: ROX_IMAGE_FLAVOR
+    value: development_build
+  dockerfile_path: operator/Dockerfile
+  inputs:
+    operator-artifacts:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/stackrox/stackrox/.
+  to: operator
+- dockerfile_path: bundle.Dockerfile
+  inputs:
+    operator-artifacts:
+      paths:
+      - destination_dir: .
+        source_path: /go/src/github.com/stackrox/stackrox/operator/.
+  to: bundle
```

Comment:
is it possible to call these something less generic? `operator-bundle`?

Reply:
How about we merge it as is, and try to rename later, so that we have continuity in operator e2e test coverage.

Reply:
I think the names are opaque to the stackrox test side as they are passed as ENV values and pushed with the names we choose to quay.io etc. But I'm ok if you want to go for it.